### PR TITLE
feat(task3): initial

### PR DESCRIPTION
**Pull request by Emigrate**

### CODE REVIEW

File: task3/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    </table>
+    <div class="b-container">
+        <div id="popuptoggle" onclick="event.stopPropagation()" class="b-popup">
+            <div class="b-popup-content">
+                <form method="post">
+                    <input type="text" placeholder="Enter Username" name="uname" required>
+                    <input type="password" placeholder="Enter Password" name="psw" required>
+                    <button type="submit">Login</button>
+                </form>
+                <button type="button" class="cancelbtn">Cancel</button>
```
Comment (Sacronux) [inline task3/index.html:68]: words must be splitted by '-'

File: task3/style.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    background: var(--buttonblue);
+    border-radius: 20px;
+    border: 3px solid silver;
+    margin-right: 10px;
+    font-size: 25px;
+    color: var(--color);
+    cursor: pointer;
+}
+
+.button-dark {
```
Comment (Sacronux) [inline task3/style.css:50]: duplication of styles `.button-login`

File: task3/style.css [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  }
+
+.b-popup{
+    display:none;
+}
+
+.b-container{
+    position: absolute;
+    width: 100%;
+    height: 2000px;
```
Comment (Sacronux) [inline task3/style.css:126]: what if the user has a display height of 4000px?

File: task3/style.css [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    width: 100%;
+  }
+
+.b-popup{
+    display:none;
+}
+
+.b-container{
+    position: absolute;
+    width: 100%;
```
Comment (Sacronux) [inline task3/style.css:125]: change % on view points

**Review by crucas — CHANGES_REQUESTED** [review]

**Review by crucas — APPROVED** [review]

**Pull request merged**